Gulpfile.js

```diff
@@ -56,6 +56,7 @@ const DEV_MODE      = 'dev' in ARGS;
 const QR_CODE       = 'qr-code' in ARGS;
 const SKIP_BUILD    = process.env.SKIP_BUILD || 'skip-build' in ARGS;
 const BROWSER_ALIAS = ARGS['browser-alias'];
+const IS_DEBUG_MODE = typeof v8debug !== 'undefined' || /--debug|--inspect/.test(process.execArgv.join(' '));
```

Comment:
Create function `getTimeout(timeout)` and move it to the `gulp/helpers/get-timeout.js` file.
Also add the !gulp statement to the .gitignore file - https://github.com/DevExpress/testcafe/pull/5724/files#diff-2f754321d62f08ba8392b9b168b83e24ea2852bb5d815d63e767f6c3d23c6ac5R3